pulsar-client/src/main/java/org/apache/pulsar/client/impl/metrics/InstrumentProvider.java

```diff
@@ -0,0 +1,59 @@
+package org.apache.pulsar.client.impl.metrics;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.api.metrics.Meter;
+import org.apache.pulsar.client.api.MetricsCardinality;
+import org.apache.pulsar.client.impl.conf.ClientConfigurationData;
+import org.apache.pulsar.common.naming.TopicName;
+
+public class InstrumentProvider {
+
+    private final Meter meter;
+    private final MetricsCardinality metricsCardinality;
+
+    public InstrumentProvider(ClientConfigurationData conf) {
+        this.meter = conf.getOpenTelemetry().getMeter("org.apache.pulsar.client");
```

Comment:
Since Pulsar's client is a library, as opposed to Pulsar Broker, we should also add the version to the meter. See: https://opentelemetry.io/docs/concepts/instrumentation/code-based/#configure-the-opentelemetry-api

Reply:
Is this using `MeterBuilder setInstrumentationVersion(String instrumentationScopeVersion);`?